plugins/com.google.cloud.tools.eclipse.appengine.deploy/plugin.xml

```diff
@@ -1,70 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
```

Comment:
plugin.xml is supposed to have at least one extension point. I believe Eclipse warns about this.